roles/nvidia-docker/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+nvidia_driver_version: 510.54
+nvidia_driver_validate: true
```

Comment:
Can we default this to false, and we can enable it explicitly if we're running outside a chroot?